docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+# BSC Staking Overview
+
+BNB Smart Chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain where staking plays a crucial role.
+[BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces native staking on BSC.
+This section covers the fundamental staking concepts and operations on BSC.
+
+## Basic Concepts
+
+### Consensus Engine
+
+BSC uses a consensus mechanism which combines DPoS and PoA for consensus, so that:
+
+* Blocks are produced by a limited set of validators.
+* Validators take turns to produce blocks in a PoA manner.
+* Validator set are elected in and out based on a staking based governance.
+
+To determinate which validators are eligible to produce blocks, that's why staking mechanism is needed.
```

Comment:
==> `"The staking is used to determine which validators are eligible to produce blocks."`

Reply:
fixed